pgx-pg-config/src/lib.rs

```diff
@@ -54,9 +54,9 @@ pub use path_methods::{get_target_dir, prefix_path};
 
 #[derive(Clone, Debug)]
 pub struct PgVersion {
-    major: u16,
-    minor: u16,
-    url: Url,
+    pub major: u16,
+    pub minor: u16,
+    pub url: Url,
```

Comment:
I've left major/minor public because I am making a comparison on `major`. For construction, I switched to `new`. I somehow missed it, even though it was right there.

Reply:
That seems like the best balance of things, yeah. Lets us add additional things to the struct if Postgres modifies their versioning scheme without breaking anyone.